<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeAttrModDialog</class>
 <widget class="QDialog" name="PeAttrModDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PE 常用属性修改</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>341</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="horizontalSpacing">
     <number>53</number>
    </property>
    <property name="verticalSpacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="oEPLabel">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Orginal Entry Point</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="oEPLineEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="imageBaseLabel">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>ImageBase</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="imageBaseLineEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="aSLRLabel">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>ASLR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="aSLRCheckBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="dEPLabel">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>DEP</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="dEPCheckBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="controlFlowGuardLabel">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Control Flow Guard</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="controlFlowGuardCheckBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="text">
    <string>说明：你正在修改 PE 文件常用属性。这里可以修改 PE 文件的 OEP，即原始入口点，以及Image Base，即默认装载地址。另外提供了ASLR、DEP、CFG的标志位的修改。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PeAttrModDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PeAttrModDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
